app/models/abstract_model.rb

```diff
@@ -131,17 +131,6 @@ def self.find_object(type, id)
     nil
   end
 
-  # Add limit to a SQL query, then pass it to find_by_sql.
-  #
-  #   sql = "SELECT id FROM names WHERE user_id = 123"
-  #   names = Name.find_by_sql_with_limit(sql, 20, 10)
-  #
-  def self.find_by_sql_with_limit(sql, offset, limit)
-    sql = sanitize_sql(sql)
-    add_limit!(sql, limit: limit, offset: offset)
-    find_by_sql(sql)
-  end
-
   # Wrap a normal SQL query in a <tt>COUNT(*)</tt> query, then pass it to
```

Comment:
Good catch, Joe.  These utilities get dug in like ticks.  Somehow we never realize when we've removed the last usage  and it's time to take them out.  I bet the last person to remove a use of this assumed it was a rails method!  Therefore it would never even occur to check to see if the method is now unused and can be removed.

Reply:
[Hey!  Wouldn't *that* be a useful cop -- catching code that is no longer used anywhere!]